atomic_reactor/core.py

```diff
@@ -236,7 +240,36 @@ def __init__(self, base_url=None, timeout=120, **kwargs):
         except AttributeError:
             # docker-py 1.x
             self.d = docker.Client(**client_kwargs)
-
+        self.retry_times = retry_times
+        self.retry_delay = DOCKER_BACKOFF_FACTOR
+        self.retry_client_statuses = DOCKER_CLIENT_STATUS_RETRY
+
+
+    def retry(function):
```

Comment:
Is this a static method? Confused by the signature not including 'self', but the function body using it.